### PR TITLE
Workflow test hardening

### PR DESCRIPTION
**Pull request by RyanLettieri**
Signed-off-by: Ryan Lettieri <ryanLettieri@microsoft.com>

# Description

## Issue reference

Please reference the issue this PR will close: #_[issue number]_

## Checklist

Please make sure you've  completed the relevant tasks for this PR, out of the following list:

* [ ] Code compiles correctly
* [ ] Created/updated tests
* [ ] Unit tests passing
* [ ] End-to-end tests passing
* [ ] Extended the documentation / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Specification has been updated / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Provided sample for the feature / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_

### CODE REVIEW

File: tests/apps/workflowsapp/app.go [hunk 1]
```diff
@@ -209,10 +212,21 @@ func main() {
 	w.RegisterWorkflow(HelloTemporalWF)
 	w.RegisterActivity(HelloTemporalAct)
 
+	for i := 0; i < 5; i++ {
```
Comment (artursouza) [inline tests/apps/workflowsapp/app.go:215]: Why 5? If just the number of workers, then it should be a const.

File: tests/dapr_tests.mk [hunk 2]
```diff
@@ -425,6 +425,10 @@ setup-test-env-temporal:
 					--set prometheus.enabled=false \
 					--set grafana.enabled=false \
 					--set elasticsearch.enabled=false \
+					--set temporalio/server=1.15.1 \
```
Comment (artursouza) [inline tests/dapr_tests.mk:428]: How will the test know to pull from GH and not from DockerHub?

**Review by artursouza — CHANGES_REQUESTED** [review]

**Comment by ItalyPaleAle** [comment]
Since this test has been deleted, this PR should probably be closed?

**Comment by RyanLettieri** [comment]
@ItalyPaleAle Yeah we can close this PR

**Pull request closed without merging**